README.md

```diff
@@ -1,44 +1,233 @@
+<div>
+    <img src="https://res.cloudinary.com/hy4kyit2a/f_auto,fl_lossy,q_70,w_50/learn/projects/quick-start-lwc-recipes-app/bb501c3216ac163958f036fb90357955_badge.png" align="left" alt="Trailhead Badge"/>
+    Learn more about using generic LWC recipes by completing the <a href="https://trailhead.salesforce.com/content/learn/projects/quick-start-lwc-recipes-app">Quick Start: Explore the LWC Recipes Sample App</a> Trailhead project.
+    <br/>
+    <br/>
+    <br/>
+</div>
 
-For example to create an Analytics template from an application run the following to see all the options available:
+> This sample application is designed to run on Salesforce Platform.
 
-'sfdx analytics:app:create --help'
+## Table of contents
 
-To view the Analytics template:
+-   [Installing the app using a Scratch Org](#installing-the-app-using-a-scratch-org): This is the recommended installation option. Use this option if you are a developer who wants to experience the app and the code.
 
-'sfdx analytics:template:list'
+-   [Tableau CRM LWC Features](#tableau-crm-lwc-features)
 
-To create an Analytics application from the template you have two options:
+-   [Tableau CRM LWC Metadata](#tableau-crm-lwc-metadata)
 
-From CLI:
+-   [Tableau CRM App Templates](#tableau-crm-app-templates)
 
-'sfdx analytics:app:create -t <templateid>'
+-   [Tableau CRM App Template Commands](#tableau-crm-app-template-commands)
 
-From Analytics Studio
+## Installing the app using a Scratch Org
 
-> Onetime: set environment variable so you always open to Analytics Page
+1. Set up your environment. Follow the steps in the [Quick Start: Lightning Web Components](https://trailhead.salesforce.com/content/learn/projects/quick-start-lightning-web-components/) Trailhead project. The steps include:
 
-export FORCE_OPEN_URL=/analytics/wave/wave.apexp
+    - Enable Dev Hub in your Trailhead Playground
+    - Install Salesforce CLI
+    - Install Visual Studio Code
+    - Install the Visual Studio Code Salesforce extensions, including the Lightning Web Components extension
 
-> Open the scratch org
+1. If you haven't already done so, authorize your hub org and provide it with an alias (**myhuborg** in the command below):
 
-sfdx force:org:open
+    ```
+    sfdx auth:web:login -d -a myhuborg
+    ```
 
-> Select 'Create' > 'App' > 'Start From Template'
+1. Clone the forcedotcom/sfdx-analytics repository:
 
-To to view the Analytics applications from the CLI:
+    ```
+    git clone https://github.com/forcedotcom/sfdx-analytics.git
+    cd sfdx-analytics
+    ```
 
-'sfdx analytics:app:list' 
+1. Create a scratch org and provide it with an alias (sfdx-analytics in the command below):
+
+    ```
+    sfdx force:org:create -s -f config/project-scratch-def.json -a sfdx-analytics
+    ```
+
+1. Push the app to your scratch org:
+
+    ```
+    sfdx force:source:push
+    ```
+
+1. Open the scratch org:
+
+    ```
+    sfdx force:org:open
+    ```
+
+## Tableau CRM LWC Features
+
+**Types**
```

Comment:
I'd include a link to https://flow.org/en/docs/types/ here since it explains the syntax and the meaning of things like `mixed` which aren't common to all typed flavors of JavaScript.

Reply:
Since lwc's main ide is currently vscode w/ the salesforce lightning-language-server (which is based on vscode's built-in typescript support), we might be better off doing typescript-style types here instead -- basically, switch `mixed` to `unknown` and remove the '`|'`s on the exact types. That might be as less strict, but more folks might grok it faster than the flow syntax.
Additionally, at some point, I am planning to add the types for the properties to the lightning-language-server so that vscode automatically knows about them (and does appropriate hover and type-checking against them)